nefertari/authentication/models.py

```diff
@@ -148,18 +148,22 @@ def get_authuser_by_name(cls, request):
             return cls.get_resource(username=username)
 
 
-def lower_strip(**kwargs):
-    return (kwargs['new_value'] or '').lower().strip()
+def lower_strip(event):
+    field = event.field
+    value = (field.new_value or '').lower().strip()
+    event.view._json_params[field.name] = value
```

Comment:
I don't like this interface. Field processors are functions meant to be written by inexperienced people. Telling them to assign values to a semi-private data structure seems unfriendly.
I would suggest a method on the event class something like `set_field_value(value, field=None)` and it could default to using the event's field (assuming one is set).